modules/twinklel10n.js

```diff
@@ -0,0 +1,84 @@
+//<nowiki>
+
+
+(function($){
+
+
+/*
+ ****************************************
+ *** twinklel10n.js: Localization module
+ ****************************************
+ */
+
+
+// Adding translations is easy, for example:
+TwinkleMessages.de = {
+	'tw-core-portlet-name': 'DE TW',
+};
+
+// Localizing twinkle is also easy, for example:
+switch( mw.config.get('wgUserLanguage') ) {
+	case 'en':
```

Comment:
@atlight This is an important part of the change that I somehow had managed not to push (accidentally put it in a separate branch).

As you can see, the default set of templates for 'en' is what EN WP uses. We can decide if we want to include any of the other translations (I vote no), but for now I am keeping a mock-translation for DE, just so you can try the demo link and see how it works (only two templates will be listed in DE interface).